src/web/modules/transactions/list.njk

```diff
@@ -133,7 +133,7 @@
       {% elseif filters.gateway_payout_id %}
         href="/transactions?page={{ set.page - 1 }}&amp;account={{ accountId }}&amp;gateway_payout_id={{ filters.gateway_payout_id }}"
       {% elseif filters.agreement_id %}
-        href="/transactions?page={{ set.page - 1 }}&amp;account={{ accountId }}&amp;agreement_id={{ filters.agreement_id }}"
+        
```

Comment:
Why was this link removed?

Reply:
Whoops! That was a mistake! Will restore!

Reply:
Fixed.